INSTALL.md

```diff
@@ -73,18 +78,32 @@ reports that Flambda is available, some further optimization options
 can be used; see the entry about `-flambda-opts` in the build guide
 for more details.
 
-Build and Installation Procedure
---------------------------------
+Choice of Build and Installation Procedure
+------------------------------------------
+
+There are two partially overlapping infrastructures available to build
+Coq. They are available through `Makefile.make` (legacy / hybrid
+build) and `Makefile.dune` (full Dune build).
```

Comment:
```suggestion
There are two partially overlapping ways to build
Coq. They are through `Makefile.make` (legacy / hybrid
build) and `Makefile.dune` (full Dune build).
```

Reply:
Not convinced by the replacement of "infrastructures" (which I used in place of the previous "build systems" which was ambiguous / created confusion) by the less specific "ways"

Reply:
IMHO, it's a little unusual, but entirely unknown, to use "infrastructure" in its plural form.  It's an undefined term, so hardly specific. It leads the reader to wonder what's in each "infrastructure", if they even care.  Also unclear what a "hybrid" build is.  And why do readers care that they overlap?

Might be clearer to say Makefile.make is the older build system.  Makefile.dune primarily uses dune, except that it invokes Makefile.make to build a few targets.

Do whatever you think is best.